packages/components/src/styles/components/tabs.scss

```diff
@@ -9,6 +9,36 @@
 
 @use "../mixins/focus-ring" as *;
 
+// Variants:
```

Comment:
for the size variants:
- usually we have the size variants _after_ the standard code (see for example the `badge.css` file)
- we tend to use a Sass map and loop over it, we could use the same approach here for consistency (again, see `badge.css` file)

https://github.com/hashicorp/design-system/blob/main/packages/components/src/styles/components/badge.scss#L37-L86